docs/creating-bots/csharp/csharp-unity-bots.mdx

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_label: 'Creating Adaptive Bots'
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Defining Bots that Adapt to the Game State
+
+Defining a Unity Bot in C# is as simple as creating a MonoBehaviour that implements the IRGBot interface.
+The IRGBot interface has no methods, but is used by the Regression Games SDK to identify bots that can be started from the in game overlay.
+
+<Tabs>
+<TabItem value="runTurn_Bot" label="Perform actions on players and NPCs" default>
+
+```csharp
+/**
+ * Outline of this Bot algorithm. For every update:
+ *  - If enough time has passed for the bot to process a new update, handle the update.
+ *  - Select a random nearby player or enemy if one exists
+ *    - Follow that entity at distance 2f
+ *    - Perform a skill targeting that entity and position
+ */
+public class RunTurnBot: MonoBehaviour, IRGBot

```

Comment:
`RunTurn` is awkward to see now since we've removed that terminology from the "early days". Want to make this something super general like "MyBot"? I think that would also better imply that the user can name the class whatever they want

Reply:
updated